buildSrc/src/main/kotlin/protokt.third-party-conventions.gradle.kts

```diff
@@ -13,6 +13,4 @@
  * limitations under the License.
  */
 
-subprojects {
-    group = "${rootProject.group}.thirdparty"
-}
+group = "com.toasttab.protokt.thirdparty"
```

Comment:
This was getting clobbered by the common-conventions plugin - we've been publishing to the wrong group since that change went in.